components/Main/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+import { View, Text, TextInput, TouchableOpacity, ScrollView } from 'react-native'
+import Item from 'components/Item'
+import './index.styl'
+import { observer } from 'startupjs'
+
+export default observer(function App () {
+
+  const [input, setInput] = useState('')
+  const[item, setItem] = useState([])
+
+  const AddTodo = () => {
+    if(input) {
+      const date = new Date()
+      setItem([...item, {
+        text : input,
+        key : Date.now(),
+        completed : false,
+        date : date.getDay() +
+          "/" + (date.getMonth() + 1) +
+          "/" + date.getFullYear()
+      }])
+    }
+    setInput('')
+  }
+
+  const EditTodo = (id, newItem) => {
```

Comment:
@TimmyByDeOverly camelCase

Reply:
@TimmyByDeOverly newItem doesn't tell use that it contains a title.

Reply:
done